src/services/dashboards/resource.js

```diff
@@ -2093,7 +2091,8 @@ export async function rollUpResourceUse(data) {
 }
 
 export async function rollUpTopicUse(data) {
-  const rolledUpTopicUse = data.topicUseResult.reduce((accumulator, topic) => {
+  const filteredTopics = data.topicUseResult.filter((topic) => topic.name !== 'Equity');
```

Comment:
Is this still needed after deleting the topic?

Reply:
yeah